bundles/org.openhab.binding.automower/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -438,7 +435,7 @@
 	<channel-type id="settingHeadlightModeType">
 		<item-type>String</item-type>
 		<label>Headlight Mode</label>
-		<description>Information about headlights</description>
+		<description>Headlight Mode</description>
```

Comment:
Remove the description if it is identical to the label (and therefor is not of any use)